esphome/components/api/client.py

```diff
@@ -18,17 +18,17 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-async def async_run_logs(config, address):
+async def async_run_logs(config: dict[str, Any], address: str) -> None:
     """Run the logs command in the event loop."""
     conf = config["api"]
+    name = config["esphome"].get("name")
```

Comment:
I wasn't sure if this could actually be missing?

Reply:
If its not possible it would be better to do

```suggestion
    name = config["esphome"]["name"]
```